### DISCUSSION TITLE
How to deploy app on vercel built with Lit Framework?

### DISCUSSION THREAD

**Discussion by akshay-verma** [vercel/community#1834] [Help] [answered]
### Description

I see `Polymer`  under Framework Preset but not Lit which is the predecessor to Polymer. 

Link to Lit Documentation - https://lit.dev/

**Comment by PonomareVlad** [comment, marked as answer]
At the moment, there is no information about the development of the _official Lit Framework_. 🥲

However, you can try other frameworks that [already integrated with Lit](https://github.com/lit/lit/tree/main/packages/labs), or you can use [this template for Vercel](https://github.com/PonomareVlad/lit-ssr-vercel).
